default.nix

```diff
@@ -93,8 +93,19 @@ stdenv.mkDerivation rec {
 
   buildFlags = [ "world" ] ++ optional buildDoc "doc-html";
 
-  installTargets = [ "install" ] ++ optional buildDoc "install-doc-html";
+  postBuild = "make byte"; # Needed by the test-suite
+  # But it is not recommended to do it in parallel with make world
```

Comment:
No -j?

Reply:
That's a solution indeed, if you are not worried about the build time.